Ginger/Ginger/ApplicationModelsLib/POMModels/PomElementsPage.xaml.cs

```diff
@@ -483,9 +483,11 @@ private void TestElementButtonClicked(object sender, RoutedEventArgs e)
                 return;
             }
 
+            ElementInfo CurrentEI = (ElementInfo)MainElementsGrid.CurrentItem;
+
             if (mSelectedLocator != null)
             {
-                mWinExplorer.TestElementLocators(new ElementInfo() { Locators = new ObservableList<ElementLocator>() { mSelectedLocator } });
+                mWinExplorer.TestElementLocators(new ElementInfo() { Path = CurrentEI.Path, Locators = new ObservableList<ElementLocator>() { mSelectedLocator } });
```

Comment:
why you create new ElementInfo object and not sending the original one? anyway the mSelectedLocator which been sent is shared between the original ElementInfo and new ElementInfo been created.